src/submission/models.py

```diff
@@ -1121,7 +1121,7 @@ def non_correspondence_authors(self):
         if self.correspondence_author:
             return self.authors.exclude(pk=self.correspondence_author.pk)
         else:
-            return self.authors
+            return self.authors.all()
```

Comment:
a bit of a minor thing:
Now that `get_initial_for_prepub_notifications` I think we could get to a situation where non correspondence authors will be added both to the `To` and `cc` because of 

> `author_initial['to'] = article.correspondence_author.email if article.correspondence_author else None`